docs/docs/additional_resources/tutorials.mdx

```diff
@@ -1,155 +1,55 @@
 # Tutorials
 
-Below are links to tutorials and courses on LangChain. For written guides on common use cases for LangChain, check out the [use cases guides](/docs/use_cases).
+### Different materials
```

Comment:
Could we keep a link to the our own internal Use Cases docs at the top level?

The Use Case docs are more likely to stay up to date as the framework evolves and will likely be expanded to cover more of the nuance of the use cases.

Reply:
After second thought... Use Cases are NOT good as Tutorials. There are many pages, like Get started / Quickstart, etc. but honestly nothing looks like tutorial. If we need the "Official" tutorial, it is TODO for now. So, U promoted Use Cases to the top level, but moved it to back, after Tutorials and Courses.